src/Heist/Extra/Splices/Pandoc/Render.hs

```diff
@@ -253,11 +253,27 @@ rpInline' ctx@RenderCtx {..} i = case i of
         w <&> \nodes ->
           [X.TextNode "“"] <> nodes <> [X.TextNode "”"]
 
+-- | Convert raw html attribute sequence into span
+--
+-- For example, this markdown: `<kbd>ctrl</kbd>`, which is in native pandoc:
+--   RawInline (Format "html") "<kbd>" : Str "ctrl" : RawInline (Format "html") "</kbd>"
```

Comment:
This looks like a `commonmark` parser bug, no? I'd expect it parse it as:

```
RawInline (Format "html") "<kbd>ctrl</kbd>"
```

Can this be fixed upstream?

Reply:
This happens with `markdown` and `gfm` too, perhaps there is an extension but I can't find it:

```ShellSession
$ pandoc --from=markdown --to=native <<< "<kbd>ctrl</kbd>"
[ Para
    [ RawInline (Format "html") "<kbd>"
    , Str "ctrl"
    , RawInline (Format "html") "</kbd>"
    ]
]

$ pandoc --from=gfm      --to=native <<< "<kbd>ctrl</kbd>"
[ Para
    [ RawInline (Format "html") "<kbd>"
    , Str "ctrl"
    , RawInline (Format "html") "</kbd>"
    ]
]
```

Reply:
A possible workaround:

```
Try this: `<kbd>space</kbd>`{=html}.
```

Not very elegant, but looks like it works.

Reply:
Nice! Though I think we should work around this issue to support the pretty syntax.

Reply:
Alright, but I wonder why the approach is:

- `AST` -> `AST`

instead of:

- `AST` -> `Html`

?

i.e., don't you want to do this logic in the rendering stage?

<img width="536" alt="image" src="https://github.com/user-attachments/assets/74bb468b-45ec-458e-924a-5542625b6db2">

It is results in much simpler code.

Reply:
It's implemented that way because the tags are adjacent, they don't form a tree, so this is more like `[AST]` -> `AST`. Not sure how to handle this case in the rendering stage because we'll have to look forward to get the inlined value.

Reply:
In that case, perhaps we should just do this AST conversion:

```
RawInline (Format "html") "<kbd>" : Str "ctrl" : RawInline (Format "html") "</kbd>"
```

to:

```
RawInline (Format "html") "<kbd>ctrl</kbd>"
```

Reply:
That looks good to me, thank you for the feedback, I'll update the PR shortly!